conftest.py

```diff
@@ -66,3 +68,61 @@ def client0():
 def saved_client0(client0):
     client0.save()
     return client0
+
+
+@pytest.fixture
+def qr():
+    return "W5P76MbdiNbXprNEnHfpcGWFp1CMF8XY"
+
+
+@pytest.fixture
+def name():
+    return "Banana"
+
+
+@pytest.fixture
+def description():
+    return "Good!"
+
+
+@pytest.fixture
+def product0(qr, name, description):
+    return Product(qr_code=qr, product_name=name, description=description)
+
+
+@pytest.fixture
+def saved_product0(product0):
+    product0.save_product()
+    return product0
+
+
+@pytest.fixture
+def supplier_product_id():
+    return 123685
+
+
+@pytest.fixture
+def quantity():
```

Comment:
I don't think we need a fixture for simple fields like those (not only quantity), the `supplier_product0` should probably be the fixture, and all of those variables can be declared in its fixture.

Reply:
This actually came from my suggestion in order to replace the use of so-called constants. Ultimately, if the value only needs to be used inside a single fixture then there's no need for another fixture, but if it is needed someplace else, then I rather have it as a fixture than as a constant (which can be modified by one test case, effecting other test cases).

Reply:
@RawanAbuleil my original comment -- that the name is too broad to be used in such context (a fixtures file used by all test modules) -- still stands.